velox/experimental/gpu/tests/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 list(APPEND CMAKE_PREFIX_PATH "${CUDAToolkit_LIBRARY_DIR}/cmake")
 find_package(CUB REQUIRED)
+find_package(Thrust REQUIRED)
```

Comment:
I am wondering if we need find_package for CUB and Thrust in the cudf CMake code? The errors we were seeing were indeed related to a mismatched CUB/Thrust but I think this file isn’t the right place to fix it. I think this is only included if building tests, and this `gpu` section of code is independent of the cuDF CMake code (it is part of the Wave GPU backend).